Turkey/TestRunner.cs

```diff
@@ -44,10 +44,12 @@ public class SystemUnderTest
 
     public class TestOutput
     {
+        public async virtual Task BeforeTesting() {}
```

Comment:
Can we make use of `AtStartupAsync` for this step?

Reply:
Thanks for pointing this out.